Code/Framework/AzCore/AzCore/Debug/PerformanceCollector.h

```diff
@@ -98,6 +103,8 @@ namespace AZ::Debug
 
         const AZStd::string& GetOutputDataBuffer() { return m_outputDataBuffer; }
 
+        const AZStd::string& GetFileExtension() { return m_fileExtension; }
```

Comment:
Nit: This method itself could be `const` as well

Reply:
Also can you can make the `GetOutputDataBuffer()` above const as well.